src/widget/tool/profileimporter.cpp

```diff
@@ -0,0 +1,85 @@
+/*
+    Copyright © 2015 by The qTox Project
```

Comment:
Wouldn't that be `2016`?

Unless it wasn't modified since 2015?

~same goes for `profileimporter.h` file